recipes/cairo/all/conanfile.py

```diff
@@ -84,20 +84,20 @@ def layout(self):
 
     def requirements(self):
         if self.options.get_safe("with_freetype", True):
-            self.requires("freetype/2.13.0")
+            self.requires("freetype/2.13.2")
         if self.options.get_safe("with_fontconfig", False):
             self.requires("fontconfig/2.14.2")
         if self.settings.os == "Linux":
             if self.options.with_xlib or self.options.with_xlib_xrender or self.options.with_xcb:
                 self.requires("xorg/system")
         if self.options.get_safe("with_glib", True):
-            self.requires("glib/2.76.3")
-        self.requires("zlib/1.2.13")
-        self.requires("pixman/0.40.0")
-        self.requires("libpng/1.6.40")
+            self.requires("glib/2.76.3", transitive_headers=True, transitive_libs=True)
+        self.requires("zlib/[>=1.2.11 <2]")
+        self.requires("pixman/0.43.4")
```

Comment:
This can't create conflicts as cairo is the only one to require this library, but will generate some missing binaries downstream

Reply:
Are you talking about the pixman bump ? it is the very motivation of this PR: pixman/0.40.0 was dropped in https://github.com/conan-io/conan-center-index/commit/ef3bb029ef69bad215c6608b371196e3f1a75f8c
To me, it seems more important to reference a maintained recipe than to prevent missing binaries, but obviously it's your call.